changelog.json

```diff
@@ -1,5 +1,17 @@
 {
   "releases": {
+    "2.6.4": [
+      "[Added] Allow users to rename and delete branches via a new context menu on branches in the branches list - #5803 #10432",
+      "[Fixed] Allow users to modify git config on a per repository basis - #9449. Thanks @say25!",
+      "[Fixed] The app is not maximized on macOS every time the user clicks on the app's icon in the dock - #11590",
+      "[Fixed] Always respect the default branch name chosen by the user - #11447",
+      "[Fixed] Notepad++ does not close when GitHub Desktop closes - #11518",
+      "[Improved] Update app icon for Windows - #11541",
+      "[Improved] Suggest emails from GitHub accounts and warn about misattributed commits in the commit message area - #11591",
+      "[Improved] Suggest emails from GitHub accounts and warn about misattributed commits in onboarding and preferences - #11561 #11566",
+      "[Improved] Remote Git operations are faster and less prone to errors on Windows - #11510",
```

Comment:
This one should be removed because right now is only enabled in beta. Or do you want me to enable it on prod for this release @billygriffin?

Reply:
We decided to enable the new trampoline for this release 🎉